obspy/seedlink/tests/test_basic_client.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+"""
+The obspy.seedlink.basic_client test suite.
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA
+
+import unittest
+
+from obspy import UTCDateTime
+from obspy.seedlink.basic_client import Client
+
+
+class ClientTestCase(unittest.TestCase):
+    def setUp(self):
+        self.client = Client("rtserver.ipgp.fr")
+
+    def test_get_waveform(self):
+        t = UTCDateTime() - 3600
+        for request in [["G", "FDF", "00", "LHN", t, t + 20],
+                        ["G", "CLF", "00", "BHZ", t, t + 10]]:
+            st = self.client.get_waveform(*request)
+            self.assertTrue(len(st) > 0)
+            for tr in st:
+                self.assertTrue(tr.id == ".".join(request[:4]))
```

Comment:
Hm, so it's about cosmetics and not about incompatibilities / unittest backwards compat etc., right? I.e. that `assertEqual(a, b)` is a bit simpler to read than `assertTrue(a == b)`? Fair point, I can change it.